documentation/common/attributes.adoc

```diff
@@ -28,6 +28,7 @@
 
 // Source and download links
 :ReleaseDownload: https://github.com/strimzi/strimzi-kafka-operator/releases[GitHub^]
+:ExampleYAMLURL: https://github.com/strimzi/strimzi-kafka-operator/tree/master/examples[Example YAML files^]
```

Comment:
I think this should not be defined upstream. Or if it is, we need to replace the link with the release version.